dev/com.ibm.ws.jsp.2.3_fat/publish/servers/jspJava8Server/server.xml

```diff
@@ -6,18 +6,17 @@
     http://www.eclipse.org/legal/epl-2.0/
```

Comment:
Update copyright date range to include 2023 since you're adding the `localConnector-1.0` feature.